addons/beehave/nodes/decorators/limiter.gd

```diff
@@ -10,7 +10,10 @@ class_name LimiterDecorator extends Decorator
 @export var max_count : float = 0
 
 func tick(actor: Node, blackboard: Blackboard) -> int:
-	var child = self.get_child(0)
+	if not get_child_count():
```

Comment:
if for whatever reason there is no child node added to this node, we should skip this behaviour (and show a configuration warning)